Classes/Repository/RepositoryViewController.swift

```diff
@@ -298,6 +298,20 @@ EmptyViewDelegate {
         present(alert, animated: trueUnlessReduceMotionEnabled)
     }
 
+    private func addDropShadow() {
+        guard let buttonBarView = buttonBarView else {
+            return
+        }
+
+        let shadowPath = UIBezierPath(rect: CGRect(origin: CGPoint(x: 0, y: buttonBarView.bounds.height),
+                                                   size: CGSize(width: buttonBarView.bounds.width, height: 1)))
+        buttonBarView.layer.masksToBounds = false
+        buttonBarView.layer.shadowColor = UIColor.black.cgColor
+        buttonBarView.layer.shadowOpacity = 1
+        buttonBarView.layer.shadowPath = shadowPath.cgPath
+        buttonBarView.layer.shouldRasterize = true
```

Comment:
Is this required?

Reply:
Not required, but since the view doesn't change, it helps to reduce shadow-drawing. Can omit it and re-add it if there was any performance issue